src/content/en/fundamentals/accessibility/00-intro/intro-four.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: shared/narrow
+title: "Screen Readers"
+description: "Introduction to screen reader software"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 4
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Accessibility is about understanding the experience of a diverse range of users. One popular type of software is the *screen reader*, a program that enables visually impaired people to use computers by reading screen text aloud in a generated voice. The user can control what is read by moving the cursor to a relevant area with the mouse or keyboard.
```

Comment:
"The user can control what is read by moving the cursor to a relevant area with the keyboard."